browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -4102,6 +4102,75 @@ If you don't accept this request, VPN will not reconnect and your internet conne
       If your current service provider is insecure, Brave uses a secure alternative DNS provider
     </message>
     <!-- End of Secure DNS Settings -->
+
+    <!-- Autofill Service strings -->
+    <message name="IDS_CITY" desc="Autofill field for city">
```

Comment:
Not sure about these translations. Are hints localised? We can't really guarantee that we will localise them precisely the same way.

Reply:
these strings are for identification. we are not 100% sure but we have to identify it from hint or label. Maybe we can find something else as well to figure out in follow up work for passwords.

Reply:
yeah, it's a question how reliable our localisations will be for proper fill